bindings/experimental/distrdf/python/DistRDF/__init__.py

```diff
@@ -47,6 +49,53 @@ def initialize(fun, *args, **kwargs):
     from DistRDF.Backends import Base
     Base.BaseBackend.register_initialization(fun, *args, **kwargs)
 
+def DeclareCppCode(code_to_declare: str) -> None:
```

Comment:
I would be lenient towards naming this `DistributeCppCode` just to be consistent with the other new functions